packages/streamline/src/components/list-items/list-item-option/list-item-option.tsx

```diff
@@ -0,0 +1,103 @@
+import { usePress } from '../../../hooks/use-press.hook';
+import { Pressable } from 'react-native';
+import { ListItemSkeleton } from '../components/list-item-skeleton';
+import { Box } from '../../../primitives/box/box';
+import ButtonIcon from '../../buttons/button-icon/button-icon';
+import LeftContent from './components/left-content';
+import { CenterContent } from './components/center-content';
+import { getBorderColor } from './list-item-option.utils';
+import { getBackgroundColor } from './list-item-option.utils';
+import { ListItemOptionProps } from './list-item-option.type';
+
+export function ListItemOption({
+  accessibilityLabel,
+  description = '',
+  header = '',
+  isSkeleton,
+  onLongPress,
+  onPress,
+  title,
+  rightOption,
+  isSelected = false,
+  disabled = false,
+  iconName,
+  type,
+}: ListItemOptionProps) {
+  const [handlePress, isResolving] = usePress({ onPress });
+
+  if (isSkeleton) {
+    return <ListItemSkeleton />;
+  }
+
+  return (
+    <Pressable
+      accessibilityLabel={
+        accessibilityLabel ?? `${header}\n${title}\n${description}`
+      }
+      disabled={disabled}
+      onPress={handlePress}
+      onLongPress={onLongPress}
+    >
+      {({ pressed }) => (
+        <>
+          <Box
+            position={'absolute'}
+            top={-2}
+            left={-2}
+            right={-2}
+            bottom={-2}
```

Comment:
In the design, the border starts outside the "regular" component size?

Reply:
this absolute view is for the shadow around 👍